### DISCUSSION TITLE
I can't install Security Onion with securityonion-2.3.170-20220922.iso

### DISCUSSION THREAD

**Discussion by ighack** [Security-Onion-Solutions/securityonion#8905] [Unsupported Versions]
I can't install Security Onion with securityonion-2.3.170-20220922.iso

that always stops at updating packages ui
![微信截图_20221012111554](https://user-images.githubusercontent.com/6670377/195242373-17c588a1-14b3-4fcf-b507-6c7039b158a2.png)

I can't get finished ui
I use https://docs.securityonion.net/en/2.3/first-time-users.html doc.

**Comment by dougburks** [comment]
Does your machine have full Internet access as shown at https://docs.securityonion.net/en/2.3/firewall.html#internet-communication?
